libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFSDKAuthPreferences.h

```diff
@@ -30,6 +30,50 @@
 #import <Foundation/Foundation.h>
 #include "SFOAuthCoordinator.h"
 NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString * const AuthPreferencesNotification NS_TYPED_ENUM NS_SWIFT_NAME(AuthPreferences.Notification);
+typedef NSString * const AuthPreferencesConstants NS_TYPED_ENUM NS_SWIFT_NAME(AuthPreferences.Constants);
+/**
+ Key identifying login host
+ */
+FOUNDATION_EXTERN AuthPreferencesConstants kSFUserAccountOAuthLoginHost;
```

Comment:
What are the swift names here? Could you maybe add comments so that a developer looking at the header file doesn't have to figure out what typedef + FOUNDATION_EXTERN does?